tlatools/org.lamport.tlatools/test/tlc2/tool/MonolithSpecTest.java

```diff
@@ -26,23 +26,36 @@
 package tlc2.tool;
 
 import static org.junit.Assert.assertTrue;
-
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
 import org.junit.Test;
 
 import tlc2.output.EC;
 import tlc2.tool.liveness.ModelCheckerTestCase;
+import util.TestPrintStream;
+import util.ToolIO;
 
 public class MonolithSpecTest extends ModelCheckerTestCase {
+        final TestPrintStream testPrintStream = new TestPrintStream();
```

Comment:
Is there any CLI formatter I could use? This is different because here we have spaces instead of tabs.

Reply:
No